to.etc.domui/src/main/java/to/etc/domui/util/exporters/PropertyExportColumn.java

```diff
@@ -47,4 +49,17 @@ public PropertyExportColumn(PropertyMetaModel<T> pmm) {
 	@Nullable @Override public T getValue(@NonNull Object in) throws Exception {
 		return m_pmm.getValue(in);
 	}
+
+	@Nullable
+	@Override
+	public Object convertValue(@Nullable Object value) throws Exception {
+		if(value instanceof Enum) {
+			if(value == null)
```

Comment:
If you have instanceof the result after it cannot be null: if value was null then that instanceof would not be true because null is not an instance of any class according to the definition of instanceof.

Reply:
You should also see that in the IDE; it should have marked that if as something problematic.